generator/src/main/java/com/scottlogic/deg/generator/validators/package-info.java

```diff
@@ -0,0 +1,6 @@
+/**
+ * Provides classes to validate the options passed on the command line.
+ *
+ * @since 1.0
+ */
+package com.scottlogic.deg.generator.validators;
```

Comment:
What is this file for?

Reply:
I'm guessing this is simply to add a javadoc to the package? I'd prefer we didn't go to this level of documentation, and leave the code as self-documenting as possible.